examples/index.json

```
@@ -992,31 +972,31 @@
         "category": "Working over the network",
         "samples": [
             {
-                "name": "Artemis ANYCAST consumer",
+                "name": "ANYCAST consumer",
```

```suggestion
                "name": "ANYCAST Consumer",
```
Shouldn't it be in uppercase to be consistent with the rest (e.g., "Basic Consumer" above).